exchanges/kraken/kraken_websocket.go

```diff
@@ -573,7 +588,13 @@ func (k *Kraken) wsProcessTrades(response []any, pair currency.Pair) error {
 			Side:         tSide,
 		}
 	}
-	return trade.AddTradesToBuffer(k.Name, trades...)
+	if tradeFeed {
+		k.Websocket.DataHandler <- trades
```

Comment:
I see we actually already have taken the step to accept trade.Data down the DataHandler in #1754
Therefore I think we should send each individual trade:
```suggestion
		for i := range trades {
			k.Websocket.DataHandler <- trades[i]
		}
```